dojo/tools/api_sonarqube/api_client.py

```diff
@@ -261,21 +261,31 @@ def get_issue(self, issue_key):
             if issue["key"] == issue_key:
                 return issue
         raise Exception(
-            f"""Expected Issue "{issue_key}", but it returned "
-            "{[x.get('key') for x in response.json().get('issues')]}."""
+            f"""Expected Issue "{issue_key}", but it returned
+            "{[x.get('key') for x in response.json().get('issues')]}."
+            full response: "
+            "{response.json()}"""
         )
 
-    def get_rule(self, rule_id):
+    def get_rule(self, rule_id, organization=None):
         """
         Get detailed information about a rule
         :param rule_id:
         :return:
         """
         rule = self.rules_cache.get(rule_id)
         if not rule:
+            request_filter = {
+                "key": rule_id
+            }
+            if organization:
+                print(organization)
```

Comment:
If this message is essential, please change it to `logger.debug(...)` with a more descriptive message. Or absolutely remove it if it was just one time debug message.

Reply:
sorry forgot to delete this. Will delete it.